backend/data/processors/conditions_preprocessing.py

```diff
@@ -432,6 +442,134 @@ def map_word_to_program_type(processed, regex_word, type):
+    
+    processed, number_of_subs = re.subn(
+    r"Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Terms 1 and 3.<br/><br/>", r"", processed, flags=re.IGNORECASE
+    )
+    if number_of_subs > 0:
+        note += "Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Terms 1 and 3."
+    
+    processed, number_of_subs = re.subn(
+    r"Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Term 3.<br/><br/>", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Term 3.<br/><br/>"
+
+    # Application and progression check notes
+    processed, number_of_subs = re.subn(
+    r"This course is by application only. Please contact the Co-op office for more information", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "This course is by application only. Please contact the Co-op office for more information"
+
+    processed, number_of_subs = re.subn(
+    r"This course is by application only.", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "This course is by application only."
+
+    processed, number_of_subs = re.subn(
+    r"Its recommended to seek a progression check prior to application.", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "It is recommended to do a progression check prior to enrolling."
+
+    processed, number_of_subs = re.subn(
+    r"This course is by application only. Please visit Business School website for more information<br/><br/>", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "This course is by application only. Please visit Business School website for more information<br/><br/>"
+
+    processed, number_of_subs = re.subn(
+    r"This course is by application only.Please visit Business School website for more information<br/><br/>", r"", processed, flags=re.IGNORECASE
+    ) 
+    if number_of_subs > 0:
+        note += "This course is by application only. Please visit Business School website for more information<br/><br/>"
+
+
+    if re.search("good academic standing", processed, flags=re.IGNORECASE) != None:
+        note += "Students must be in good academic standing."
+    
+    processed = re.sub(
+    r"be in good academic standing", r"", processed, flags=re.IGNORECASE
+    )
+    
+    processed, number_of_subs = re.subn(
+    r"in their final year of a single or double Commerce degree", r"", processed, flags=re.IGNORECASE
```

Comment:
You could probably make the substitution string in some of these be "COMM#" like you have in line 544 - the note is still needed since this condition doesn't check what year they're in but it's still an implicit requirement